### PR TITLE
Whitelist the views we want, show less buttons

### PR DESCRIPTION
**Pull request by cryos**
This change adds some of the custom application support from ParaView
(thanks to Utkarsh) in order to simplify the user interface. We should
add more views, and possibly more buttons, but this shows a simple
starting point and we can test it more widely.

### CODE REVIEW

File: tomviz/ViewFrameActions.cxx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "ViewFrameActions.h"
+
+#include <pqRenderView.h>
+#include <pqSpreadSheetView.h>
```
Comment (mathturtle) [inline tomviz/ViewFrameActions.cxx]: Are these used?

**Comment by mathturtle** [comment]
Tested and it works.

File: tomviz/ViewFrameActions.cxx (hunk 1 above)
Comment (cryos) [inline tomviz/ViewFrameActions.cxx]: They were in my testing, but I yanked that code. I can remove them.

**Comment by cryos** [comment]
Thanks for testing, I will merge this and see if it looks good in the binaries later.

**Pull request merged**